move super shell rendering to logger

eed3si9n opened the pull request:
Fixes #4583
Ref https://github.com/sbt/util/pull/196

eed3si9n committed:
move super shell rendering to logger

Fixes #4583
Ref https://github.com/sbt/util/pull/196

typesafe-tools commented:
A validation involving this pull request <a href="https://jenkins.scala-sbt.org:8592/job/sbt-validator/1507/console">is in progress...</a>

typesafe-tools commented:
[//]: # (Refs: pull/4604/head-develop-develop-develop-pull/196/head)
The validator <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/1507/">has checked</A> the following projects, tested using dbuild, projects built on top of each other.

Project             | Reference            | Commit
:------------------ | :------------------- | :----------------
  sbt               | <A HREF="https://github.com/sbt/sbt/pull/4604">pull/4604/head      </A> | sbt/sbt@0162209b04cda3201f454f10cea84ecbf09d18f9
  zinc              | <A HREF="https://github.com/sbt/zinc/tree/develop">develop             </A> | sbt/zinc@6c0e9cbc84ecc08dede9fbe3c2df4ca34761de19
  io                | <A HREF="https://github.com/sbt/io/tree/develop">develop             </A> | sbt/io@c2668e77443e2a6fd7c33da706c299bf5a795e69
  librarymanagement | <A HREF="https://github.com/sbt/librarymanagement/tree/develop">develop             </A> | sbt/librarymanagement@f073dde812848ef443619578f55e290f3863bcaa
  util              | <A HREF="https://github.com/sbt/util/pull/196">pull/196/head       </A> | sbt/util@bc376c27df392065bf193d83d4f851acfc78007c

:white_check_mark: The result is: ***SUCCESS***
<A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild?sbt=pull/4604/head&zinc=develop&io=develop&librarymanagement=develop&util=pull/196/head&deploy_to_bintray=false&run_sbt_community=false">(restart)</A>

eed3si9n committed:
Undo cursor tweaks around prompt

eed3si9n committed:
Make loggers synchronized

This is to workaround for "[success]" logs displaying after the prompt is displayed.

eed3si9n committed:
Util 1.3.0-M6
